accessibility/common-remediations.md

```diff
@@ -263,6 +442,23 @@ We _don’t_ do this:
 
 ### Implied headings
 
+#### What's the problem?
+
+CSS has been used to visually style text so that it looks like a heading, but there are no semantic heading tags (i.e. `<h1>`, `<h2>`, `<h3>`, `<h4>`, `<h5>`, `<h6>`).
+
+#### Who's affected by the problem?
+
+* Screen reader users
+* Search engines
+
+#### Why's it a problem?
+
+Semantic tags have meaning. When semantic tags are absent from your document, it makes less sense. This is a severe issue for screen reader users, who need proper semantic structure in order to understand how your document is structured. 
+
+Search engines also use heading levels to get a sense of the information that your page contains. If your document omits semantic tags, search engines are less able to parse the information in your document and your pages could be penalised in SERPs, or be less discoverable to search engine users. 
```

Comment:
Make `SERPs` a link to `https://en.wikipedia.org/wiki/Search_engine_results_page`?